tests/__init__.py

```diff
@@ -239,6 +239,11 @@ class Str2Int(Dict[str, int]):
     pass
 
 
+@dataclass
+class OptTuple:
+    x: Optional[Tuple[int, ...]] = None
```

Comment:
What is the element_type if the resulting ListConfig?

Reply:
The `element_type` is `int` :)

Reply:
FYI in the following case we also have `element_type==int`:
```python
@dataclass
class OptTuple:
    x: Optional[Tuple[int, str]]
```
Something to watch out for...